keps/sig-cli/3659-kubectl-apply-prune/README.md

```diff
@@ -479,26 +394,26 @@ treat this manifest as invalid and return an error.
 Although ApplySet parent objects can (in theory) be of any type, for both performance and simplicity, we choose to limit supported types to the following:
 - ConfigMap
 - Secret
-- custom resources, where the CRD registering them is labeled with `applyset.k8s.io/role/parent` (The value is currently ignored, but implementors should set an empty value to be forwards-compatible with future evolution of this convention.)
+- custom resources, where the CRD registering them is labeled with `applyset.kubernetes.io/is-parent-type` (The value is currently ignored, but implementors should set it to "true" to be forwards-compatible with future evolution of this convention.)
 
 This list may be extended in the future as use cases are presented. Keeping it minimal has significant benefits for the performance of any commands that list the parent objects themselves. Specific tools may further limit the types they support, but any tooling designed to identify all ApplySets must consider the exact list set out above.
 
 While a purpose-made cluster-scoped CRD is a logical choice, the specification has no opinion on this, so as to accommodate the many and various choices existing tools have already made. In choosing types to support as parents, tools should consider what permissions their target users typically have on their clusters; for instance, they may not have permissions to install CRDs.
 
 #### Labels and annotations
 
-ApplySets MUST also have a “parent object” in the cluster. The ApplySet object MUST be labeled with:
+ApplySet parent objects MUST carry the following label:
 
 ```yaml
-applyset.k8s.io/id: <applysetid> # REQUIRED
+applyset.kubernetes.io/id: <applysetid> # REQUIRED
 ```
 
-The `applyset.k8s.io/id` label is what makes the object an ApplySet parent object. Its value MUST match the value of `applyset.k8s.io/part-of` on the member objects (see [ApplySet Member Objects](#labels)), and MUST comply with the naming constraints outlined in [ApplySet Naming](#applyset-naming).
+The `applyset.kubernetes.io/id` label is what makes the object an ApplySet parent object. Its value MUST match the value of `applyset.kubernetes.io/part-of` on the member objects (see [ApplySet Member Objects](#labels)), and MUST use the format outlined in [ApplySet Identification](#applyset-identification).
 
-Additionally, ApplySet parents MUST be labelled with:
+Additionally, ApplySet parents MUST be annotated with:
```

Comment:
This might have been an uncaught mistake on my part, but in the real code we are currently checking an annotation. While we don't have any reason to query on this, it might be useful? Only to a limited degree, given that the value includes version info. I could go either way.